### PR TITLE
Update neomodel and make it heroku deployable

### PR DESCRIPTION
**Pull request by nossila**
Updates django and neomodel and also added a `Procfile` to make it easier to deploy to heroku.

How to test:

1. Create an [sandbox database](https://sandbox.neo4j.com/), make sure to select **Paradise Papers by ICIJ** under **Pre-Built Data**, copy the credentials: username, password and bolt URL, you are going to need that later.
2. Create an heroku app, lets call it `paradise-papers` as an example.
    1. Go to its settings and add the following config vars:
        1. `ALLOWED_HOST` with your heroku app URL, in this example: `paradise-papers.herokuapp.com`
        2. `DATABASE_URL` with the credentials from your sandbox database in the following format: `bolt://user:password@ip:port`
3. Clone the repository `git clone git@github.com:silverlogic/paradise-papers-django.git`
4. `cd paradise-papers-django`
5. Change to this branch `git checkout GGP-409-heroku`
6. Add your heroku app's git URL as a remote `git remote add heroku https://git.heroku.com/paradise-papers.git`
7. Push to heroku to deploy the app `git push heroku master`
8. Go to your app URL http://paradise-papers.herokuapp.com

### CODE REVIEW

**Comment by whatSocks** [comment]
@nossila can you update the readme as well with the updated packages

**Comment by whatSocks** [comment]
Running app locally following readme instructions, got this error:
```
Watching for file changes with StatReloader
Performing system checks...

Traceback (most recent call last):
  File "/Users/dddd/Documents/_projects/TSL/paradise-papers-django/paradise_papers_search/venv/lib/python3.8/site-packages/django/apps/registry.py", line 155, in get_app_config
    return self.app_configs[app_label]
KeyError: 'admin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
    ```

File: paradise_papers_search/paradise_papers_search/settings/base.py [hunk 1]
```diff
@@ -26,9 +25,6 @@
 
 INSTALLED_APPS = [
     'django_neomodel',
-    'django.contrib.admin',
```
Comment (whatSocks) [inline paradise_papers_search/paradise_papers_search/settings/base.py:29]: i had to re-add this stuff to get the app to run

File: paradise_papers_search/paradise_papers_search/settings/base.py [hunk 2]
```diff
@@ -40,7 +36,6 @@
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
```
Comment (whatSocks) [inline paradise_papers_search/paradise_papers_search/settings/base.py:43]: had to re-add this to get app to run

**Comment by whatSocks** [comment]
Second issue - app runs but when I try to use the search fields or drowpdowns it 500
```
django.core.exceptions.ImproperlyConfigured: settings.DATABASES is improperly configured. Please supply the ENGINE value. Check settings documentation for more details.
[31/Jan/2021 23:42:16] "GET /fetch/datasource HTTP/1.1" 500 24276
Internal Server Error: /fetch/jurisdictions
Traceback (most recent call last):
```

**Comment by nossila** [comment]
@whatSocks can you review again please?

**Review by whatSocks — APPROVED** [review]

**Pull request merged**